app/models/answer.rb

```diff
@@ -0,0 +1,23 @@
+class Answer < ActiveRecord::Base
+  belongs_to :user
+  belongs_to :question
+  has_many :comments, as: :response, dependent: :destroy
+  has_many :votes, as: :tally, dependent: :destroy
+
```

Comment:
good work with validations on all of your associations and uniqueness constraints!